### DISCUSSION TITLE
Closer integration with Quarkus Core/Quarkiverse/Quarkus Platform

### DISCUSSION THREAD

**Discussion by lwitkowski** [quarkiverse/quarkus-resteasy-problem#116] [General]
**What do we want to achieve?**
1. We want this extension to be listed on https://code.quarkus.io/
2. `./mvnw quarkus:add-extension -Dextensions="quarkus-resteasy-problem"` should just work
3. We don't want end users to be forced to specify (and bump) this extension's version in their `pom.xml` - we want to be a part of `quarkus-universe-bom`

**How to get there?**

https://quarkus.io/blog/quarkiverse/ - this requires changing group/artefact id and migrate project to quarkiverse github org

[Quarkus Platform](https://quarkus.io/guides/platform) - some 3rd party extensions are part of quarkus universe bom, but are not in quarkiverse though: https://github.com/datastax/cassandra-quarkus

3rd option is to become integral part of Quarkus core, there's [ongoing discussion on dev mailing list](https://groups.google.com/g/quarkus-dev/c/N-RcsYKJzpo/m/k1xfi4m9BwAJ)

**Comment by hamburml** [comment]
May I know the status? Would love to see this in Quarkiverse or directly in Quarkus

**Comment by lwitkowski** [comment]
We're in the process of integrating with Quarkiverse, so closing this discussion.
